src/Execute/ExecutionManager.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {globalExecutorArray} from '../Backend/Backend';
+import {Executor} from '../Backend/Executor';
+import {Device} from './Device';
+
+interface DeviceExecutorSet {
+  [key: string]: Set<Executor>;
+}
+
+class ExecutionManager {
```

Comment:
Q. How about 'ExecutorManager' here?

Reply:
hmmm It will handle both `Executor` and `Device`, which both related to on `Execution`. so I name like this.

Reply:
Now that 'Device' class has 'Executors' as a member, it can be a DeviceManager. :-D

Reply:
um.... I think this implementation only handle about Device on this manager, so I accept `DeviceManager`. I will fix this.